### PR TITLE
Ankit/add composio agent

### PR DESCRIPTION
**Pull request by ankit-v2-3**
- Add Composio Agent and Tool.

### CODE REVIEW

**Comment by ashish-spext** [comment]
1. We should have Text Component of composio response for v1, for cases like below.
image


2. We should make easy for users to add tools which don't require API Keys (e.g hackernews)
Maybe let's add them in .env.sample as commented line?

File: backend/director/agents/composio.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        },
+    },
+    "required": ["task"],
+}
+
+
+class ComposioAgent(BaseAgent):
+    def __init__(self, session: Session, **kwargs):
+        self.agent_name = "composio"
+        self.description = f'The Composio agent is used to run tasks related to apps like {os.getenv("COMPOSIO_APPS", ["GMAIL"])} '
```
[the remark this answers is not in the document]
Comment (ankit-v2-3) [reply backend/director/agents/composio.py]: Yeah, removed gmail

**Comment by ankit-v2-3** [comment]
Response from Composio is not readable, so not adding into Text component.

**Comment by ankit-v2-3** [comment]
Added additional LLM call to format Composio response. Can you please test the experience?

**Review by ashish-spext — CHANGES_REQUESTED** [review]

**Review by ashish-spext — APPROVED** [review]

**Pull request merged**